### ISSUE TITLE
Document package override option

### ISSUE THREAD

**Issue by imaustink** [bit-docs/bit-docs-generate-html#46]
Somewhere, we need to document this feature.
https://github.com/bit-docs/bit-docs-generate-html/pull/45#pullrequestreview-56695951

**Comment by imaustink** [comment]
@leoj3n any thoughts?

**Comment by leoj3n** [comment]
Yes we should document this. Perhaps you could tell me more about what it does and I can help you document it.
